server/planning/planning_notifications.py

```diff
@@ -48,3 +61,172 @@ def notify_assignment(self, target_user=None, target_desk=None, target_desk2=Non
                     continue
                 add_activity(ACTIVITY_UPDATE, can_push_notification=True, resource='assignments', msg=message,
                              notify=[member.get('user')], **data)
+
+        # determine if a Slack Bot has been configured
+        if slack_client_installed and app.config.get('SLACK_BOT_TOKEN'):
+            args = {'token': app.config.get('SLACK_BOT_TOKEN'), 'target_user': target_user, 'target_desk': target_desk,
+                    'target_desk2': target_desk2, 'message': _get_slack_message_string(message, data)}
+            self._notify_slack.apply_async(kwargs=args)
+
+    def user_update(self, updates, original):
+        """
+        Superdesk user has been updated
+
+        If the update includes the Slack username then check that that user can be found in the Slack workspace
+
+        :param updates:
+        :param original:
+        :return:
+        """
+        if app.config.get('SLACK_BOT_TOKEN'):
```

Comment:
Might be good to check `slack_client_installed` here as well

Reply:
Yep will do!